angular/src/app/landing/landingpage.component.ts

```diff
@@ -171,12 +171,15 @@ export class LandingPageComponent implements OnInit, AfterViewInit {
                 // id not found; reroute
                 console.error("No data found for ID=" + this.reqId);
                 metadataError = "not-found";
-                //   this.router.navigateByUrl("/not-found/" + this.reqId, { skipLocationChange: true });
             }
             else{
-                // Get metrics 
-                if(this.inBrowser)
-                    this.getMetrics();
+                // Get metrics when edit is not enabled. Otherwise display "Metrics not available"
+                if(this.inBrowser){
+                    if(this.routerParamEditEnabled)
```

Comment:
This should test `this.cfg.editEnabled` instead of `this.routerParamEditEnabled` since the latter only determines if the `editEnabled` query parameter was used.  The former properly indicates that we are running on the publishing side.